docs/editors/vim.md

```diff
@@ -58,6 +45,19 @@ let g:lsc_auto_map = {
     \}
 ```
 
+Run `:PlugInstall` to install the plugin. If you already have `vim-lsc`
```

Comment:
Run PlugInstall (or the relevant command for your plugin manager)

Reply:
It already says "assuming vim-plug above" and highlights vundle works too. Nobody has complained about these instructions so far so I guess it's been fine. The main improvement in this PR was to move all `~/.vimrc` config into one code listing.